libs/design-system/src/lib/Components/LandingHeader/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { ButtonLink } from '../Button';
+import { GradientName, LandingHeaderContainer } from './LandingHeaderContainer';
+
+export type LandingHeaderProps = {
+  title: string;
+  gradient: GradientName;
+  callout: string;
+  description: string;
+  buttonText: string;
+  buttonUrl: string;
+};
+
+export function LandingHeader({
+  buttonText,
+  buttonUrl,
+  callout,
+  description,
+  gradient,
+  title,
+}: LandingHeaderProps) {
+  return (
+    <LandingHeaderContainer gradient={gradient}>
+      <div className="flex max-w-6xl flex-col justify-items-stretch px-4 py-12 md:px-20 md:py-20">
+        <h1 className="text-h1 mb-6 max-w-full overflow-hidden text-ellipsis !text-4xl md:!text-7xl">
+          {title}
+        </h1>
+        <div className="flex flex-row items-stretch rounded-lg bg-primary-gray-100/30">
+          <div className="px-5 py-3 md:basis-1/2 md:px-20 md:py-12">
+            <h2 className="text-h2 mb-2 line-clamp-1 md:mb-3 md:line-clamp-2">
```

Comment:
Should we remove the line clamp here? Only 1.5 words can be seen on mobile

Reply:
Sure happy to remove that! The reason I added that was because the Figma design had a placeholder title of "Featured callout here two lines." - so I thought that was indicating that this should be limited to 2 lines. And the mobile design uses the title "One liner" so I thought that was indicating just a single line on mobile. But definitely simpler to get rid of the clamp entirely.